pkg/types/baremetal/platform.go

```diff
@@ -61,4 +61,14 @@ type Platform struct {
 
 	// DNSVIP is the VIP to use for internal DNS communication
 	DNSVIP string `json:"dnsVIP"`
+
+	// BootstrapMachineOSImage is a URL to override the default RHCOS image
```

Comment:
s/RHCOS/machine OS/

Eventually this will also be FCOS

Reply:
Fixed. Thanks.